### PR TITLE
Disable windows variant

### PR DESCRIPTION
**Pull request by gsoltis**
- windows is showing a "The batch file cannot be found." error that we are not currently looking into
 - only run the tests if this PR changes something relevant to them.

### CODE REVIEW

File: .github/workflows/pr-js-tests.yml [hunk 1]
```diff
@@ -30,4 +30,4 @@ jobs:
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
 
-      - run: pnpm -- turbo run test --filter=!cli --color
+      - run: pnpm -- turbo run test  --filter=...[${{ env.GITHUB_BASE_REF }}] --filter=!cli --color
```
Comment (gaspar09) [inline .github/workflows/pr-js-tests.yml:33]: `env.GITHUB_BASE_REF` is empty

https://github.com/vercel/turborepo/actions/runs/3107719853/jobs/5036137265#step:5:12

Comment (gaspar09) [reply .github/workflows/pr-js-tests.yml:33]: Is that intentional?

Comment (gaspar09) [reply .github/workflows/pr-js-tests.yml:33]: I believe these are not accessed from `env`
https://docs.github.com/en/actions/learn-github-actions/environment-variables#:~:text=C%3A%5Chostedtoolcache%5Cwindows-,Note%3A,read%20during%20workflow%20processing%20using%20the%20%24%7B%7B%20github.ref%20%7D%7D%20context%20property.,-Detecting%20the%20operating

Comment (gsoltis) [reply .github/workflows/pr-js-tests.yml:33]: hmm, not intentional.

**Comment by gsoltis** [comment]
@gaspar09 good catch. I _think_ I've got the right variable now

**Review by gaspar09 — APPROVED** [review]
🤠 🚀

**Pull request merged**